src/components/card/ThumbnailCard.jsx

```
@@ -1,7 +1,7 @@
 import React from "react";
 import { nameElipse } from "../../utils/common/nameElipse";
 
-const ThumbnailCard = ({ id, image, title, onClick }) => {
+const ThumbnailCard = ({ image, title, onClick }) => {
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)